### PR TITLE
fix #79481 hocvientruyentranh.net

### LINKED ISSUE #79481: hocvientruyentranh.net
### Issue URL (Ads)
 [https://hocvientruyentranh.net/truyen/5040/years](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhocvientruyentranh.net%2Ftruyen%2F5040%2Fyears)
### Comment
Username: `@Uegou`
> Ads pop up sometimes when I open this website so I close and open again, the problem seems to be fix.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 beta 1
Browser: | Edge 89.0.774.68 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (180), <br>Disable cache for third-party requests, <br>Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Hide your Referrer from third-parties (hocvientruyentranh.net), <br>Hide your User-Agent (), <br>Hide your IP address (209.141.53.254)
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> EasyList Italy, <br/>AdBlockID, <br/>AdGuard Russian, <br/>AdGuard Japanese
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/beta/adguard-wot/1.0/wot.user.js,<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
Referrer: | [https://hocvientruyentranh.net/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhocvientruyentranh.net%2F)
Other extensions used: | window defender, Chongluadao extension

### PR DESCRIPTION
**Pull request by wout-codes**
#79481
Ads, popup overlay fixed in ABPVN List.

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -235,6 +235,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79481
+hocvientruyentranh.net#%#//scriptlet("abort-on-property-read", "popup_link")
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/general_extensions.txt:239]: I can't reproduce popups and I can't find popup_link variable/function in site's code

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/general_extensions.txt:239]: @wout-codes 
Please check do we still need this rule?

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:239]: Yes, I can still reproduce sometimes, not everytime.

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/general_extensions.txt:239]: @AdguardTeam/filters-maintainers please take a look

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/general_extensions.txt:239]: I still can't reproduce popups and I can't find popup_link variable/function in site's code
@wout-codes could you please make screenshot with popup_link in code?

Comment (stanislav-atr) [reply EnglishFilter/sections/general_extensions.txt:239]: @Sergey-Lyapin, popup not reproduced on my side either.

**Pull request closed without merging**